draft-ietf-jsonpath-base.md

```diff
@@ -1101,7 +1101,7 @@ logical-and-expr  = basic-expr *(S "&&" S basic-expr)
                       ; binds more tightly than disjunction
 
 basic-expr        = paren-expr /
-                    relation-expr /
+                    comp-expr /  ; comparison
```

Comment:
We might call it "comparison-expr" and get rid of the comment.

Reply:
I toyed with that. Adopted.